docs/catalog/cloud/Design for independent scaling of logical components.md

```diff
@@ -0,0 +1,39 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- kubernetes
+- role:software-engineer
+- role:cloud-engineer
+- size:medium
+---
+
+# Design for independent scaling of logical components
+
+## Description
+A microservice architecture may reduce the compute resources required, as it allows for independent scaling of its logical components and ensures they are scaled according to the demand.
+
+## Solution
+
+Use [Dapr](https://dapr.io) Framework or other cloud native projects to help you separate your application functionality into different microservices, to allow independent scaling of its logical components.
+
+## SCI Impact
+`SCI = (E * I) + M per R`
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Regarding the SCI equation, designing for independent scaling of logical components will impact:
+
+- `E`: By optimizing the most consuming application components, we reduce the amount of energy consumed by the application globally.
```

Comment:
Consider changing to "By optimizing the scaling of the most consuming application components..." to emphasize that this pattern is about separating your application into logical pieces to efficiently support scaling